src/javascript/app/base/binary_pages.js

```diff
@@ -50,7 +50,8 @@ const TNCApproval             = require('../pages/user/tnc_approval');
 const VideoFacility           = require('../pages/user/video_facility');
 
 // ==================== static ====================
-// const Charity            = require('../../static/pages/charity');
+/*
+const Charity            = require('../../static/pages/charity'); */
```

Comment:
why this change?

Reply:
Mainly, because I thought it look prettier because it's aligned other imports

Reply:
that's fine, better go with consistency and use `//` for single-line comments

Reply:
sure